integration/test-integration/src/test/resources/dockerCompose/apim-in-common-network-docker-compose.yaml

```diff
@@ -7,10 +7,10 @@ services:
     image: wso2am/wso2am:4.1.0-alpine
     user: root
     ports:
-      - "9763:9763"
-      - "9443:9443"
-      - "5672:5672"
-      - "8243:8243"
+      - "9764:9764"
+      - "9444:9444"
+      - "5673:5673"
+      - "8244:8244"
```

Comment:
We don't need to expose 8244 port. It is the Synapse port. Again we can avoid exposing 9764 port as it is HTTP port.

Reply:
I'll remove 8244. As for 9764, we are using it for an health check with `/service/Version`. Where even if the cert fails this returns a 200.

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2869/commits/4d4400d4a048460039a1216e4cc2cef970c81e06